lib/carousel_slider.dart

```diff
@@ -54,12 +54,12 @@ class CarouselSlider extends StatefulWidget {
       required this.itemBuilder,
       required this.options,
       this.disableGesture,
-      CarouselController? carouselController,
+      CarouselSliderController? carouselController,
       Key? key})
       : items = null,
         _carouselController = carouselController != null
             ? carouselController as CarouselControllerImpl
-            : CarouselController() as CarouselControllerImpl,
+            : CarouselSliderController() as CarouselControllerImpl,
```

Comment:
I think that you need rename `CarouselControllerImpl` to `CarouselSliderControllerImpl` also. @jamesldr

Reply:
Is becomes more consistent and cohesive

Reply:
I agree, I'm submitting a new commit with this change, thanks!